packages/@expo/cli/src/start/server/metro/MetroBundlerDevServer.ts

```diff
@@ -131,8 +133,30 @@ export class MetroBundlerDevServer extends BundlerDevServer {
         path.relative(appDir, filepath.replace(/\.[tj]sx?$/, '.js'))
       );
       if (contents) {
+        let src = contents.src;
```

Comment:
Add a TODO about merging this with the other source map treatment in the future https://github.com/expo/expo/blob/0dffdb15d9f3b3053458023b54153cb7a5f92b73/packages/%40expo/metro-config/src/serializer/serializeChunks.ts#L422-L439